### ISSUE TITLE
Add support for Redis 7 in the reid integration

### ISSUE THREAD

**Issue by Alphayeeeet** [elastic/integrations#10199]
The docs says it was only tested up to redis 3.X.

If the integration works for never versions, it should be mentioned in the doc.

**Comment by victor-chan-groundswell** [comment]
FWIW, I tried to add the integration to an AWS managed redis cache, with version 7.1.0 (elasticache).... sincce there's no native AWS integration yet except tweaking cloudwatch metrics... and while the integration seems to work, and documents related to that cache gets indexed into elasticsearch, the dataset is completely empty.

**Comment by Alphayeeeet** [comment]
Yes, pretty similar as our usecase. We use the redis integration for the redis instance of our Gitlab servers, which use newer versions as currently supported. To allow to fully monitor such applications, extended version support would be necessary. 

@andrewkroh Do you know, if there are any updates from the team, regarding to implement this in the beats?

**Comment by andrewkroh** [comment]
The redis integration doesn't fall under my purview. But it looks like the package lacks automated integration tests with redis. In my opinion, ideally the integration be tested against the set of actively maintained redis versions. For example, NATS has a `variants.yml` that lists a few images that the package is tested with.

https://github.com/elastic/integrations/blob/3d2478d019212b53a8959182ccafa947e2422fb5/packages/nats/_dev/deploy/variants.yml

It could be that the underlying source code is already integration tested elsewhere (i.e. in elastic/beats metricbeat), and has been tested with newer versions, but the docs haven't been updated in the package. The elastic/obs-infraobs-integrations team can probably give us a definitive answer.

**Comment by Alphayeeeet** [comment]
+1

**Comment by Alphayeeeet** [comment]
If resolved already, please mention in the docs and close this issue.
